src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -908,6 +909,8 @@ def power_plants_status_quo(scn_name="status2019"):
     con = db.engine()
     session = sessionmaker(bind=db.engine())()
     cfg = egon.data.config.datasets()["power_plants"]
+    year = int(scn_name[-4:])
```

Comment:
`scenario_date_max = f"{scn_name[-4:]}-12-31 23:59:00"`

Reply:
Implemented in 341db612b04a6fcd966a876ce78dfebde3ed31da